ParsePush/build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'com.android.library'
+
+group = 'com.parse'
+version = '1.6.1'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.3.1'
+    }
+}
+
+dependencies {
+    compile "com.google.android.gms:play-services:3.1.+"
+    compile files("libs/classes.jar")
+}
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion "23.0.1"
+
+    defaultConfig {
+        minSdkVersion 10
+        targetSdkVersion 23
+        versionCode 1
+        versionName project.version
+    }
+
+    lintOptions {
+        abortOnError false
+    }
+}
+
+android.libraryVariants.all { variant ->
+    task("jar${name.capitalize()}", type: Jar) {
+        dependsOn variant.javaCompile
+        from variant.javaCompile.destinationDir
+
+        manifest {
+            attributes(
+                "Bundle-Name": 'parse-android-push',
```

Comment:
not so sure about the `Bundle-Name`, right now in android, we name them like
`parse-XXX-android` or `xxx-android`. Check [here](http://mvnrepository.com/artifact/com.parse).